TestExistingNewsletterView test failing in CircleCI

## Description

See: https://circleci.com/api/v1.1/project/github/mozilla/bedrock/48385/output/103/0?file=true&allocation-id=633d7739f62e3807bfc1b66d-0-build%2F5310B29B

This just started happening today, seemingly out of the blue. So far we can't replicate the failures locally at all.

Specific errors seem to be:

`TypeError: '>=' not supported between instances of 'MagicMock' and 'int'`

and

```
>       request, template_name, context = render.call_args[0]
E       ValueError: not enough values to unpack (expected 3, got 2)
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -4,7 +4,9 @@ orbs:
 jobs:
   test_py:
     docker:
-      - image: cimg/python:3.9
+      - image: cimg/python:3.9.13
+        # 3.9.13 is a temporary downpin while we shake out test failures that happen only on 3.9.14
+        # See https://github.com/mozilla/bedrock/issues/12203
         auth:
           username: $DOCKERHUB_USERNAME
           password: $DOCKERHUB_PASSWORD

